tests/scripts/run-xunit-perf.cmd

```diff
@@ -44,12 +46,18 @@ setlocal ENABLEDELAYEDEXPANSION
 
   rem find and stage the tests
   set /A "LV_FAILURES=0"
-  for /R %CORECLR_PERF% %%T in (*.%TEST_FILE_EXT%) do (
-    call :run_benchmark %%T || (
-      set /A "LV_FAILURES+=1"
+
+  if defined SLICE_DIR (
+    for /R %SLICE_DIR% %%S in (slice*.txt) do (
```

Comment:
Just trying to understand - I see the JSON file that contains the configuration for each slice.  Does this get converted to intermediate text files?

Reply:
Yeah, that's what I have the python script doing. It parses the JSON and then generates slice<number>.txt files which are read by run-xunit-perf.cmd. I couldn't think of another way to get the folders out of the json file in batch.